modules/end-user-portal/src/Core/ServiceCollectionExtensions.cs

```diff
@@ -65,8 +65,8 @@ public static IServiceCollection ConfigureApplicationServices(this IServiceColle
         services.AddAntiforgery(options =>
         {
             options.SuppressXFrameOptionsHeader = true;
-            options.Cookie.HttpOnly = false;
-            options.Cookie.SecurePolicy = CookieSecurePolicy.Always;
+            options.Cookie.HttpOnly = true;
+            options.Cookie.SecurePolicy = CookieSecurePolicy.SameAsRequest;
```

Comment:
So I think this is right having reviewed a number of articles which suggest that setting httpOnly = true will prevent client side scripting attacks.